src/sql/workbench/contrib/dataExplorer/browser/dataExplorerViewlet.ts

```diff
@@ -93,18 +88,6 @@ export class DataExplorerViewPaneContainer extends ViewPaneContainer {
 		return 400;
 	}
 
-	getSecondaryActions(): IAction[] {
-		let menu = this.menuService.createMenu(MenuId.DataExplorerAction, this.contextKeyService);
```

Comment:
Are you planning on bringing back the contribution point at some point? If we aren't (which we should consider doing since I'm not sure we have a good use case for that being available) then we should clean up the contribution point as well